File: problem_solving/compare-the-triplets/answer.py
def compareTriplets(a, b):
    aScore = 0
    bScore = 0
    for i in range(len(a)):
        if a[i] > b[i]:
            aScore += 1
        elif a[i] < b[i]:
            bScore += 1
    return [aScore,bScore]

File: problem_solving/compare-the-triplets/answer.js
function compareTriplets(a, b) {
    let aScore = 0;
    let bScore = 0;
    a.forEach((item, ind)=>{
        if (item > b[ind]) {
            aScore += 1;
        } else if (item < b[ind]) {
            bScore += 1;
        }
    });   
    return [ aScore, bScore];
}
